test/DAREditor/MainWindow.xaml.cs

```diff
@@ -23,16 +24,56 @@ namespace DAREditor
     /// </summary>
     public partial class MainWindow : Window
     {
-        readonly ViewModel m_viewModel = new ViewModel();
+        readonly ViewModel _viewModel = new ViewModel();
 
         public MainWindow()
         {
-            this.DataContext = m_viewModel;
+            this.DataContext = _viewModel;
             InitializeComponent();
+            ActualRichTextBox.TextChanged += ActualRichTextBox_TextChanged;
+            ExpectedRichTextBox.TextChanged += ExpectedRichTextBox_TextChanged;
+        }
+
+        bool statusExpected = true;
```

Comment:
>statusExpected [](http://example.com/codeflow?start=13&length=14)

nit: rename these two to something like `_expectedTextValid` and move to the top of the class.